fdbclient/include/fdbclient/FDBTypes.h

```diff
@@ -1675,8 +1675,8 @@ struct Versionstamp {
 
 	template <class Ar>
 	void serialize(Ar& ar) {
-		int64_t beVersion;
-		int16_t beBatch;
+		uint64_t beVersion;
```

Comment:
Is this intentional?

Reply:
Yes both version and batch (which these are set to) are `uint64_t`

Reply:
`Versionstamp` has been there for a long time and has been used in many critical components, am not sure if this change breaks any existing usages, or, I would be careful making change here unless it is necessary.

Reply:
This is actually new Versionstamp was recently created by AJ for this specific usecase, the old Versionstamp class was renamed to something else. Currently its only my code that uses this new class